doc/src/manual/mathematical-operations.md

```diff
@@ -331,7 +331,18 @@ comparison is undefined. It is strongly recommended not to use expressions with
 as printing) in chained comparisons. If side effects are required, the short-circuit `&&` operator
 should be used explicitly (see [Short-Circuit Evaluation](@ref)).
 
-### Operator Precedence
+### Elementary Functions
+
+Julia provides a comprehensive collection of mathematical functions and operators. These mathematical
+operations are defined over as broad a class of numerical values as permit sensible definitions,
+including integers, floating-point numbers, rationals, and complexes, wherever such definitions
```

Comment:
Perhaps "complex numbers" rather than "complexes"? Or by "complexes" do you mean composite types of some form?